docs/user/ppl/functions/json.rst

```diff
@@ -60,3 +58,281 @@ Example::
     | json scalar string  | "abc"                           | "abc"                   |
     | json empty string   |                                 | null                    |
     +---------------------+---------------------------------+-------------------------+
+
+JSON_OBJECT
+----------
+
+Description
+>>>>>>>>>>>
```

Comment:
please add `Version` and `Limitation` section for all new added functions.
Limitation: Only works when `plugins.calcite.enabled=true`

Reply:
Done.